### PR TITLE
Sign Windows builds

### PR DESCRIPTION
**Pull request by GuillaumeRx**

### CODE REVIEW

File: forge.config.js [hunk 1]
```diff
@@ -17,9 +17,15 @@ module.exports = {
   makers: [
     {
       name: '@electron-forge/maker-squirrel',
-      config: {
-        name: 'signer'
-      }
+      config: process.env.SHOULD_SIGN
```
Comment (FrederikBolding) [inline forge.config.js:20]: Wouldn't this work?
`{ name: 'signer', ...(process.env.SHOULD_SIGN ? { certificateFile: bla } : {}) }`

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: forge.config.js (hunk 1 above)
Comment (Mrtenz) [reply forge.config.js:20]: Is the conditional needed at all? Wouldn't it just ignore it if `WINDOWS_PFX_FILE` is `undefined`?

Comment (FrederikBolding) [reply forge.config.js:20]: 🤷‍♂️

Comment (GuillaumeRx) [reply forge.config.js:20]: Yes you're right @Mrtenz !

**Review by Mrtenz — APPROVED** [review]

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**